docs/release-notes.txt

```diff
@@ -8,6 +8,39 @@
  Version 0.14
 **************
 
+Version 0.14.4
+==============
+
+**Release Date:** March 17, 2021
+
+**New Features**
+
+-  API: Add a new endpoint exposing historical cluster resource
+   allocation. Currently, information about experiment workloads
+   (training, checkpoints, and validations) is included. Use the
+   :ref:`rest-api` for more information.
+
+-  Introduce a stopping-based variant of
```

Comment:
We probably want some categorization of these lines.  i.e. "Scheduler:"

Reply:
Do we have a master list of categorizations anywhere? I'd prefer not to do an ad-hoc process every time (I actually thought I deleted the API category above, but I think I forgot to save my changes), and I'm not quite sure they actually improve readability or comprehension.

Reply:
We don't have a master list.  I think they do improve the docs in the sense that people can more easily skip over the notes that they are not interested in.